css-layout-api/Overview.bs

```diff
@@ -70,6 +70,8 @@ urlPrefix: https://www.w3.org/TR/CSS21/; type:dfn
 urlPrefix: https://html.spec.whatwg.org/#; type: dfn
     text: structuredserializeforstorage
     text: structureddeserialize
+urlPrefix: https://webidl.spec.whatwg.org/; type: dfn
+    url: #is-a-platform-object; text: is a platform object
```

Comment:
Delete these lines, WebIDL is in the linking database and that anchor is linkable normally.

(It's just not exported, so you'd need to specify the spec manually in link-defaults right now, tho I'm fixing that in <https://github.com/whatwg/webidl/pull/1427>)